src/pages/Bridge/RedeemForm/index.tsx

```diff
@@ -91,7 +91,7 @@ const RedeemForm = (): JSX.Element | null => {
   const wrappedTokenAmount = watch(WRAPPED_TOKEN_AMOUNT);
 
   const [dustValue, setDustValue] = React.useState(BitcoinAmount.zero());
-  const [status, setStatus] = React.useState(STATUSES.IDLE);
+  const [status, setStatus] = React.useState<any>(STATUSES.IDLE);
```

Comment:
Do we really need this `any`? We do not allow `any` for strict typing.

Reply:
We do not need it, a better solution would be to use the type of STATUSES. That would work great if STATUSES were an enum. It still works regardless